src/test/regress/expected/isolation_database_cmd_from_any_node.out

```diff
@@ -0,0 +1,235 @@
+citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+step s2-commit: COMMIT;
+
+starting permutation: s1-enable-create-db-prop s1-create-testdb1 s1-begin s2-begin s1-acquire-citus-adv-oclass-lock-with-oid-testdb1 s2-acquire-citus-adv-oclass-lock-with-oid-testdb1 s1-commit s2-commit s1-drop-testdb1
+step s1-enable-create-db-prop: SET citus.enable_create_database_propagation TO ON;
+step s1-create-testdb1: CREATE DATABASE testdb1;
+step s1-begin: BEGIN;
+step s2-begin: BEGIN;
+step s1-acquire-citus-adv-oclass-lock-with-oid-testdb1: SELECT citus_internal_acquire_citus_advisory_object_class_lock((SELECT CASE WHEN substring(version(), '\d+')::integer < 16 THEN 25 ELSE 26 END AS oclass_database), 'testdb1');
+citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+step s2-acquire-citus-adv-oclass-lock-with-oid-testdb1: SELECT citus_internal_acquire_citus_advisory_object_class_lock((SELECT CASE WHEN substring(version(), '\d+')::integer < 16 THEN 25 ELSE 26 END AS oclass_database), 'testdb1'); <waiting ...>
+step s1-commit: COMMIT;
+step s2-acquire-citus-adv-oclass-lock-with-oid-testdb1: <... completed>
+citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+step s2-commit: COMMIT;
+step s1-drop-testdb1: DROP DATABASE testdb1;
+
+starting permutation: s1-enable-create-db-prop s1-create-testdb1 s2-enable-create-db-prop s2-create-testdb2 s1-begin s2-begin s1-acquire-citus-adv-oclass-lock-with-oid-testdb1 s2-acquire-citus-adv-oclass-lock-with-oid-testdb2 s1-commit s2-commit s1-drop-testdb1 s2-drop-testdb2
+step s1-enable-create-db-prop: SET citus.enable_create_database_propagation TO ON;
+step s1-create-testdb1: CREATE DATABASE testdb1;
+step s2-enable-create-db-prop: SET citus.enable_create_database_propagation TO ON;
+step s2-create-testdb2: CREATE DATABASE testdb2;
+step s1-begin: BEGIN;
+step s2-begin: BEGIN;
+step s1-acquire-citus-adv-oclass-lock-with-oid-testdb1: SELECT citus_internal_acquire_citus_advisory_object_class_lock((SELECT CASE WHEN substring(version(), '\d+')::integer < 16 THEN 25 ELSE 26 END AS oclass_database), 'testdb1');
+citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+step s2-acquire-citus-adv-oclass-lock-with-oid-testdb2: SELECT citus_internal_acquire_citus_advisory_object_class_lock((SELECT CASE WHEN substring(version(), '\d+')::integer < 16 THEN 25 ELSE 26 END AS oclass_database), 'testdb2');
+citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+step s1-commit: COMMIT;
+step s2-commit: COMMIT;
+step s1-drop-testdb1: DROP DATABASE testdb1;
+step s2-drop-testdb2: DROP DATABASE testdb2;
```

Comment:
a shared `teardown` with `DROP DB IF EXISTS` for all the databases would be nice. Right now the cleanup distracts a bit from the actual test

Reply:
Such a teardown block would mean an implicit transaction block and DROP DATABASE cannot be executed in a transaction block, so unfortunately cannot have a common teardown block :/

teardown
{
    DROP DATABASE IF EXISTS testdb1;
    DROP DATABASE IF EXISTS testdb2;
    DROP DATABASE IF EXISTS db1;
}

Reply:
ugh, I forgot about that. Let's leave as is then.